<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор паролей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>581</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>Сгенерированные пароли появятся здесь</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_generate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать пароли</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exportpasswords">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>720</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Экспорт паролей в файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exportsettings">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>720</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить настройки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_importsettings">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>720</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить настройки</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_characterset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>581</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Набор символов</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_excludedchars">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>561</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Символы, которые не должны использоваться</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_excludedchars">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Исключаемые символы:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_uppercase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Заглавные буквы (A-Z)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_includedchars">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>561</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Символы, которые могут использоваться</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_lowercase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Строчные буквы (a-z)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_specialchars">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Специальные символы (!@#$...)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_includedchars">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Включаемые символы:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_preinstalledsets">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Предустановленные наборы:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_digits">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Цифры (0-9)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_preinstalledsets">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string>Пользовательский</string>
     </property>
     <item>
      <property name="text">
       <string>Пользовательский</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Только буквы</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Буквы и цифры</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Буквы, цифры и спец. символы</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_mainsettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Основные настройки</string>
    </property>
    <widget class="QSpinBox" name="spinBox_number">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_min">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Минимальная:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_length">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Длина пароля:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_max">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Максимальная:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_max">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>42</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_min">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_number">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество паролей:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_advsettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>581</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Расширенные настройки</string>
    </property>
    <widget class="QCheckBox" name="checkBox_morereadablepassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Генерировать более читаемый пароль (две согласные-гласная)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_onlyuppercase">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Только верхний регистр</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_readablepassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Генерировать читаемый пароль (чередование согласная-гласная)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_preventrepeat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Предотвращать повторение символов</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxrepeats">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Макс. количество последовательных повторений:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_maxrepeats">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>42</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_onlylowercase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Только нижний регистр</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
